internal/runners/packages/install.go

```diff
@@ -39,13 +43,35 @@ func (a *Install) Run(params InstallRunParams, nsType model.NamespaceType) (rerr
 		nsTypeV = &nsType
 	}
 
+	ts := params.Timestamp.Time
+	if ts == nil {
+		latest, err := model.FetchLatestTimeStamp()
+		if err != nil {
+			return errs.Wrap(err, "Unable to fetch latest Platform timestamp")
+		}
+
+		commitID, err := commitmediator.Get(a.prime.Project())
+		if err != nil {
+			return errs.Wrap(err, "Unable to get commit ID")
+		}
+
+		_, atTime, err := model.FetchCheckpointForCommit(commitID)
```

Comment:
Nit: a model function like `FetchTimeStampForCommit` might be nice. It could return the `time.Time` value of the timestamp for us.